src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql

```diff
@@ -506,9 +509,15 @@ select
     t.name,
     t.start_date,
     t.end_date,
+    f.nj_student_tier,
 from base_roster as b
 left join
     expanded_terms as t
     on cast(b.academic_year as int) = t.academic_year
     and b.expected_test = t.name
     and b.region = t.region
+left join
+    {{ ref("int_reporting__student_filters") }} as f
+    on cast(b.academic_year as int) = f.academic_year
+    and b.student_number = f.student_number
+    and {{ union_dataset_join_clause(left_alias="b", right_alias="f") }}
```

Comment:
@GabyRangelB pls add and f.iready_subject = 'Reading' to the join (filter is subject-dependent)